### PR TITLE
Add nodejs

### PR DESCRIPTION
**Pull request by flowerett**

### CODE REVIEW

**Comment by flowerett** [comment]
setup on staging successful:
```
deploy@chronos:/home/ubuntu$ node -v
v5.7.1
```

File: conichi-nodejs/recipes/default.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
+#
+# Cookbook Name:: conichi-nodejs
+# Recipe:: default
+#
+
+bash 'Configure the server timezone as CET' do
```
Comment (cheredon) [inline conichi-nodejs/recipes/default.rb]: you know what to do :grinning:

**Comment by flowerett** [comment]
need review :)

**Comment by cheredon** [comment]
gut

**Comment by rubyengineer** [comment]
:ok:

**Pull request merged**